larray/core/array.py

```diff
@@ -2942,12 +2942,14 @@ def with_total(self, *args, **kwargs):
 
         Parameters
         ----------
-        args
-        kwargs
-        op : aggregate function
-            Defaults to `sum()`.
-        label : scalar value
-            label to use for the total. Defaults to "total".
+        *args : int or str or Axis or Group or any combination of those, optional
+            Axes or groups along which to compute the aggregates. Passed groups should be named. 
+            Defaults to aggregate over the whole array.
+        **kwargs:
+            * op : aggregate function, optional 
+                Defaults to `sum()`.
+            * label : scalar value, optional 
+                label to use for the total. Applies only on aggregated axes, not groups. Defaults to "total".
```

Comment:
please do not use this syntax (`**kwargs` : then listing the kwargs). It does not render nicely in sphinx.

Reply:
Do I remove `**kwargs` or unident kwargs below?